md6/src/consts.rs

```diff
@@ -0,0 +1,61 @@
+/// MD6 constants related to standard mode of operation
+pub type Md6Word = u64;
+pub type Md6ControlWord = u64;
+pub type Md6NodeID = u64;
+
+/// Maximum stack height
+pub const MD6_MAX_STACK_HEIGHT: usize = 29;
+/// Maximum number of rounds
+pub const MD6_MAX_R: usize = 255;
+/// Large so that MD6 is fully hierarchical
+pub const MD6_DEFAULT_L: usize = 64;
+
+/// Number of bits in a word
+pub const MD6_W: usize = 64;
+/// Size of compression output in words
+pub const MD6_C: usize = 16;
+/// Size of compression input block in words
+pub const MD6_N: usize = 89;
+
+// These five values give lengths of the components of compression
+// input block; they should sum to MD6_N.
+/// Q words in a compression block (>= 0)
+pub const MD6_Q: usize = 15;
+/// Key words per compression block (>= 0)
+pub const MD6_K: usize = 8;
+/// Words for unique node ID (0 or 64/w)
+pub const MD6_U: usize = 64 / MD6_W;
+/// Words for control word (0 or 64/w)
+pub const MD6_V: usize = 64 / MD6_W;
+/// Data words per compression block (> 0)
+pub const MD6_B: usize = 64;
```

Comment:
I don't think we need these separate `MD6_*` constants. Just inline the values to `W`, `C`, `N`, etc. constants defined below.